### ISSUE TITLE
YAML chart parsing issue

### ISSUE THREAD

**Issue by truonghuynguyen** [plantuml/plantuml#528]
#### Given this plantuml:
```
@startyaml
level1:
    level1.1: "data: value"
    level1.2: "data:value"
@endyaml
```

#### It generates the following diagram.
![image](https://user-images.githubusercontent.com/28770410/114442829-d4e81c80-9b81-11eb-8fb9-1941a2f64786.png)

#### Expected:
Notice the space in `level1.1` key, `level1.1` should be rendered as `level1.2`.

**Comment by kirchsth** [comment]
(found it in the release notes) fixed with V1.2021.5

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now fixed.
- Thanks @truonghuynguyen, @kirchsth for the report.
- Thanks PlantUML team for your work.
	
Regards.
